app/internal/report/report.go

```diff
@@ -0,0 +1,98 @@
+package report
+
+import (
+	"encoding/binary"
+	"time"
+
+	"app/internal/util"
+)
+
+const (
+	// UploadKeyLen is the length, in bytes, of an UploadKey
+	UploadKeyLen = 16
+
+	pendingReportsCollection = "pending_reports"
+
+	// 3 days - the period of during which a token is valid and may be verified.
```

Comment:
Are these constants inherent to the protocol? Perhaps they should live in a config that can be changed without recompiling?

Reply:
I definitely want to do that eventually, but for prototyping's sake, it's not a priority. If you want to tackle that, though, feel free! https://github.com/covid19risk/covidwatch-cloud-functions/issues/32